### PR TITLE
Zaeem khan

### PR DESCRIPTION
**Pull request by zaeem-khan**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -3,7 +3,6 @@ target/
 !.mvn/wrapper/maven-wrapper.jar
 !**/src/main/**/target/
 !**/src/test/**/target/
-logs/
```
Comment (Neha-Bhatia551) [inline .gitignore:6]: Can you add this. back? so that logs are ignored when we commit?

File: logs/app2.log [hunk 2]
```diff
@@ -13,3 +13,73 @@
 2023-09-30 08:29:03,460 INFO o.s.b.StartupInfoLogger [main] Starting LibraryprojectApplicationTests using Java 17.0.7 with PID 28600 (started by hafiz in D:\School work\Grad_stuff\SE 452\libraryproject)
 2023-09-30 08:29:03,479 INFO o.s.b.SpringApplication [main] No active profile set, falling back to 1 default profile: "default"
 2023-09-30 08:29:07,598 INFO o.s.b.StartupInfoLogger [main] Started LibraryprojectApplicationTests in 4.414 seconds (process running for 5.808)
+2023-10-01 16:20:10,397 INFO o.s.b.StartupInfoLogger [main] Starting LibraryprojectApplicationTests using Java 17.0.7 with PID 20836 (started by hafiz in D:\School work\Grad_stuff\SE 452\libraryproject)
```
Comment (Neha-Bhatia551) [inline logs/app2.log]: Can you remove logs when u commit, so please delete this and commit again

File: .gitignore (hunk 1 above)
Comment (zaeem-khan) [reply .gitignore:6]: just confirmed that it's still in the gitignore

File: logs/app2.log (hunk 2 above)
Comment (zaeem-khan) [reply logs/app2.log]: removed

**Review by Neha-Bhatia551 — APPROVED** [review]

**Pull request merged**